linux/tools.Dockerfile

```diff
@@ -69,6 +69,8 @@ COPY ./linux/powershell/Invoke-PreparePowerShell.ps1 linux/powershell/Invoke-Pre
 
 # Remove su so users don't have su access by default.
 RUN rm -f ./linux/Dockerfile && rm -f /bin/su
+# cleanup tmp files left behind by installation
+RUN find /tmp/ -mindepth 1 -delete
```

Comment:
I was wondering if we can clean up such stuff as soon as it is created in any `RUN` step? So there is no cruft in any layer. 

As in this is a nice addition from security POV, but not from the size POV.

Reply:
i would love to, but the files seem to come many differents `RUN` steps, and it would make the dockerfile quite hard to read. Although i will check - maybe it's only a small number of RUN that need the cleanup.